src/test/java/marquez/db/models/OwnerRowTest.java

```diff
@@ -23,7 +23,7 @@
 public class OwnerRowTest {
   private static final UUID ROW_UUID = UUID.randomUUID();
   private static final Instant CREATED_AT = Instant.now();
-  private static final String NAME = "test owner";
+  private static final String NAME = "test connectionUrl";
```

Comment:
Same comment around renaming

Reply:
Please see PRs #394, #387, and #384, where this is fixed.